schemas/pxm-manifest-0.5.0.json

```diff
@@ -1,4 +1,6 @@
 {
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://mapbox.com/schemas/pxm-manifest-0.5.0.json",
```

Comment:
According to https://spacetelescope.github.io/understanding-json-schema/basics.html#declaring-a-json-schema, the `$schema` and `id` items are standard practice.

These URLs don't exist yet, of course.

Reply:
@sgillies can you point to the GitHub url instead to `mapbox.com` ?